tests/Operation/WatchFunctionalTest.php

```diff
@@ -130,6 +131,100 @@ function(stdClass $command) use (&$commands) {
         $this->assertSame($expectedCommands, $commands);
     }
 
+    public function testResumeBeforeReceivingAnyResultsIncludesStartAtOperationTime()
+    {
+        $operation = new Watch($this->manager, $this->getDatabaseName(), $this->getCollectionName(), [], $this->defaultOptions);
```

Comment:
I originally inserted a document before starting the change stream, but that lead to a strange behavior. The initial `aggregate` command would not find the document (of course), but it's response would include an `operationTime` time based on the document's insertion since it was the most recent entry in the oplog.

When we kill and resume the cursor in the subsequent test activity, that operation time would be used for `startAtOperationTime` in the resuming `aggregate` and the document would show up.

I was under the impression that using the operation time from `aggregate` was intended to prevent extra documents from showing up from before the launch point of the original change stream, but perhaps this is a known edge case.

Perhaps @daprahamian, @kevinAlbs, or @ShaneHarvey can chime in here.

Reply:
@ShaneHarvey mentioned that he and @behackett talked about this some time ago and concluded that it was expected behavior (if no other write operation on the server occurs between the `insert` and `aggregate` nor does enough time elapse for a periodic no-op entry to be created).

They tossed around the idea of having the driver manually increment the operation time returned by `aggregate`, but nothing came of that.

Reply:
Yeah I just talked with Bernie, incrementing the operation time does not really solve the problem. We can fix the issue in SPEC-1122.

Reply:
Aye, SPEC-1122 ("Update startAtOperationTime during change stream iteration" for those without access) would help provided we're not resuming because the first `getMore` has failed; however, I'll admit that this is a fairly contrived edge case more likely to show up in a test suite than real life.